src/main/java/ua/toe/tac/tic/TicTacToe.java

```diff
@@ -13,7 +13,8 @@
 import static ua.toe.tac.tic.enums.Direction.*;
 
 public class TicTacToe {
-
+    private int countX = 15;
```

Comment:
Did not get the reason of adding it here. Could you please add more code to make it usable

Reply:
Sorry.
Of course, there is no point in this line.
The children and I figured out how to make branches and forks on github. I used your project for demo.